### ISSUE TITLE
Allow suggestions to be inserted and removed one by one

### ISSUE THREAD

**Issue by Mauker1** [Mauker1/MaterialSearchView#25]
As for the release 1.1.1 and below, it's only possible to add all the search suggestions at once. As it's only possible to remove them all.

On the new release, it'll be possible to add and remove suggestions one by one.

**Comment by x4080** [comment]
Hi, I tried onQueryTextChange, I remove all suggestions and add all suggestions that is relevant for the query text, and I found some problems :

1. onQueryTextChange is called twice for each key entered
2. Remove and add all suggestions is slow (maybe must remove and add in another thread)
3. the new suggestion is not displayed until new key is entered, can there be an api to refresh the listview display ? Or is it already there

Thanks
